pontoon/base/tests/test_views.py

```diff
@@ -557,10 +557,9 @@ def test_entity_filters(self):
             else:
                 params['status'] = filter_
 
-            with patch('pontoon.base.models.Entity.objects.{}'.format(filter_name), return_value=Entity.objects.all()) as filter_mock:
+            with patch('pontoon.base.models.Entity.objects.{}'.format(filter_name), return_value=getattr(Entity.objects, filter_name)()) as filter_mock:
```

Comment:
The trick is here. Previously we used `Entity.objects.all()` to fake `Entity.objects.missing()` and friends, but this no longer works since we don't return QuerySets, but Q objects.